### PR TITLE
Build Sphinx Docs

### PR DESCRIPTION
**Pull request by luc10921**
**Related issue**
#452 

**Summary or solution description**
Added neo3-boa builtin function to the docs.

**Platform:**
 - OS: Windows 10 x64
 - Python version: Python 3.8

### CODE REVIEW

**Comment by luc10921** [comment]
@lock9, could you review if the descriptions of the variables of the following classes are good?
https://github.com/CityOfZion/neo3-boa/blob/52494f055956fe02c54b9f4e80a3e8eaf538bce6/boa3/builtin/interop/blockchain/block.py
https://github.com/CityOfZion/neo3-boa/blob/52494f055956fe02c54b9f4e80a3e8eaf538bce6/boa3/builtin/interop/blockchain/transaction.py
https://github.com/CityOfZion/neo3-boa/blob/52494f055956fe02c54b9f4e80a3e8eaf538bce6/boa3/builtin/interop/contract/contract.py
https://github.com/CityOfZion/neo3-boa/blob/52494f055956fe02c54b9f4e80a3e8eaf538bce6/boa3/builtin/interop/runtime/notification.py
https://github.com/CityOfZion/neo3-boa/blob/52494f055956fe02c54b9f4e80a3e8eaf538bce6/boa3/builtin/interop/storage/storagecontext.py
https://github.com/CityOfZion/neo3-boa/blob/52494f055956fe02c54b9f4e80a3e8eaf538bce6/boa3/builtin/interop/storage/storagemap.py

**Comment by lock9** [comment]
Hi @ixje , could you look at this? I don't know if I have enough knowledge to review this properly.

**Comment by luc10921** [comment]
The ones I was the most in doubt of are Block.transaction_count, Contract.update_count, and StorageMap.prefix, because I couldn't find a good definition on Neo's docs.

File: boa3/builtin/interop/runtime/notification.py [hunk 1]
```diff
@@ -4,6 +4,17 @@
...
 
 class Notification:
+    """
+    Represents a notification.
+
+    :var script_hash: the script hash of the notification sender
+    :vartype script_hash: UInt160
+    :var event_name: the notification's name
+    :vartype event_name: str
+    :var state: the object representing the notification content, which can be of any type such as value, string, array,
```
Comment (ixje) [inline boa3/builtin/interop/runtime/notification.py:14]: I don't know how this is represented on the boa side, but on the C# side `state` is an VM type Array ([this](https://github.com/neo-project/neo-vm/blob/master/src/neo-vm/Types/Array.cs)) holding other VM types.
Maybe @meevee98 can have a look at this description

**Review by ixje — COMMENTED** [review]
I left a couple of comments based on how I recently documented them in the `dev` branch in `mamba`. I couldn't really comment the storage map as that doesn't look familiar to me.

File: boa3/builtin/interop/runtime/notification.py (hunk 1 above)
Comment (meevee98) [reply boa3/builtin/interop/runtime/notification.py:14]: On boa it's represented as a `tuple` (as specified in the following line)

```suggestion
    :var state: a tuple value storing all the notification contents.
```

**Review by lllwvlvwlll — APPROVED** [review]
This looks good.  Now we can link users to the docs instead of the code for method reference.

**Pull request merged**